src/qml/AboutPage.qml

```diff
@@ -55,6 +55,11 @@ ItemPage {
                 value: about.kernelVersion
             }
 
+            StandardItem {
+                key: qsTr("Memory Size")
```

Comment:
Could this be perhaps changed to "RAM Size"? I think it'd be more clear - some people still use "memory" to refer to storage medium, most commonly on mobile devices.

Reply:
updated